docs/volume_annotation/import_export.md

```diff
@@ -16,4 +16,19 @@ After finishing the annotation of a volume layer with a fallback layer, the comb
 Icon to open the materialize volume annotation dialog
 ///
 
-This button opens up a dialog that starts a long-running job which will materialize the volume annotation.
\ No newline at end of file
+This button opens up a dialog that starts a long-running job which will materialize the volume annotation.
+
+## Restricting magnifications
+
+WEBKNOSSOS allows data annotation in different magnifications.
+Restricting the available magnifications can greatly improve the performance when annotating large structures, such as nuclei, since the volume data does not need to be stored in all quality levels. 
+How to read: Mag 1 is the most detailed, 4-4-2 is downsampled by factor 4 in x and y, and by factor 2 in z.
+When annotating large structures, consider restricting the available magnifications in this specific annotation or in this volume annotation layer.
```

Comment:
I think, there is a slight misunderstanding regarding how things can be restricted. let me try to clear this up a bit:

```suggestion
When annotating large structures, consider restricting the available magnifications in the volume annotation layer(s).
The restriction can be defined when a new volume layer is created (either when an annotation is created or when a new volume layer is added to an annotation).
```